<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>analysisMaster</class>
 <widget class="QWizard" name="analysisMaster">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>532</width>
    <height>414</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Мастер анализа данных</string>
  </property>
  <widget class="QWizardPage" name="wizardPage1">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Анализ данных</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>10</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QListWidget" name="listWidget">
      <item>
       <property name="text">
        <string>Наивный Байесовский классификатор</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Метод опорных векторов</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Дерево решений</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Метод k-ближайших соседей</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>setType()</slot>
 </slots>
</ui>
